server/src/commandLine.mjs

```diff
@@ -69,6 +72,12 @@ const parsed = nopt(knownOpts, shortHands, process.argv, 2);
 const httpPort = parsed.httpPort || config.app.httpPort;
 const socketPort = parsed.socketPort || config.app.socketPort;
 
+// --- novalidate method overrides
+if(!config.dotConfig.validateFlag || parsed.novalidate ){
```

Comment:
I like this better than what I had in mind, but seeing it in front of my eyes now, I think we should change dotConfig.validateFlag to dotConfig.validateMethodOverrides so the name matches the config setting/property.

Reply:
Done